docs/design-proposals/cloud-edge-communication/cloud-edge-application-communication-framework.md

```diff
@@ -0,0 +1,17 @@
+# Arktos Cloud-edge Application Communication Framework Overview
+
+## Motivation
+
+As the next generation cloud platform architecture, arkto need to consider the edge cloud scenario in the architecture and scheme. In the edge cloud scenario, the computing nodes will be pulled to the edge site to run, some delay sensitive applications, such as manufacturing systems and real-time trading systems, need arktos cloud platform to provide cloud services locally to achieve local access or local data processing, as well as to achieve real-time response, which requires these cloud services to be pulled to the edge and requires arktos to provide secure communication framework between cloud and edges.
```

Comment:
scenario in the architecture and scheme. -> scenario as part of the overall architecture.